docs/book/driver-deployment/deploying_csi_with_zones.md

```diff
@@ -0,0 +1,314 @@
+
+![REGION_AND_ZONES](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/REGION_AND_ZONES.png)
+
+The sample workflow creates zones per cluster and per data center.
+
+Procedure
+
+- Create Zones Using vSphere Tags
+  - You can use vSphere tags to label zones in your vSphere environment.
+- Enable Zones for the CCM and CSI Driver
+  - Install the CCM and the CSI driver using the zone and region entries.
+  
+### Create Zones Using vSphere Tags
+
+You can use vSphere tags to label zones in your vSphere environment.
+
+The task assumes that your vCenter Server environment includes three clusters, cluster1, cluster2, and cluster3, with the node VMs on all three clusters. In the task, you create two tag categories, k8s-zone and k8s-region. You tag the clusters as three zones, zone-a, zone-b, and zone-c, and mark the data center as a region, region-1.
+
+#### Prerequisites
+
+Make sure that you have appropriate tagging privileges that control your ability to work with tags. See vSphere Tagging Privileges in the vSphere Security documentation.
+
+**Note:** Ancestors of node VMs, such as host, cluster, and data center, must have the ReadOnly role set for the vSphere user configured to use the CSI driver and CCM. This is required to allow reading tags and categories to prepare nodes' topology.
+
+#### Procedure
+
+1. In the vSphere Client, create two tag categories, k8s-zone and k8s-region.
+
+   For information, see [Create, Edit, or Delete a Tag Category](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenterhost.doc/GUID-BA3D1794-28F2-43F3-BCE9-3964CB207FB6.html) in the vCenter Server and Host Management documentation.
+
+2. In each category, create appropriate zone tags.
+
+   For information on creating tags, see Create, Edit, or Delete a Tag in the vCenter Server and Host Management documentation.
+
+    <table>
+    <thead>
+    <tr>
+    <th>Categories</th>
+    <th>Tags</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+    <td>k8s-zone</td>
+    <td>zone-a<br>zone-b<br>zone-c</td>
+    </tr>
+    <tr>
+    <td>k8s-region</td>
+    <td>region-1</td>
+    </tr>
+    </tbody>
+    </table>
+
+3. Apply corresponding tags to the data center and clusters as indicated in the table.
+
+   For information, see Assign or Remove a Tag in the vCenter Server and Host Management documentation.
+
+    <table>
+    <thead>
+    <tr>
+    <th>vSphere Objects</th>
+    <th>Tags</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+    <td>datacenter</td>
+    <td>region-1</td>
+    </tr>
+    <tr>
+    <td>cluster1</td>
+    <td>zone-a</td>
+    </tr>
+    <tr>
+    <td>cluster2</td>
+    <td>zone-b</td>
+    </tr>
+    <tr>
+    <td>cluster3</td>
+    <td>zone-c</td>
+    </tr>
+    </tbody>
+    </table>
+
+## Enable Zones for the vSphere CPI and CSI Driver <a id="enable_zones_for_vsphere_cpi_and_csi"></a>
+
+Install the vSphere CPI and the CSI driver using the zone and region entries.
+
+Procedure
```

Comment:
Can you highlight this.
Procedure ==> #### Procedure

Reply:
done